.eslintrc.js

```diff
@@ -1,12 +1,11 @@
 module.exports = {
+    "root": true,
     "env": {
         "browser": true,
         "es6": true
     },
-    "extends": [
-        "prettier",
-    ],
     "ignorePatterns": ["dist/**/*.ts", "packages/contracts/hardhat"],
```

Comment:
This exclusion for `hardhat` is no longer working :/

Reply:
How are determining this?